ocrd/utils.py

```diff
@@ -21,11 +21,16 @@
     'polygon_from_points',
     'is_string',
     'concat_padded',
-    'safe_filename'
+    'safe_filename',
+    'is_local_filename',
+    'unzip_file_to_dir',
+    'abspath'
```

Comment:
abspath → absolute_path?

Reply:
But it is just a small wrapper around `os.abspath`.

Reply:
Yeah, I thought so.